### ISSUE TITLE
Facebook login fails from "Already have account" page

### ISSUE THREAD

**Issue by lukemeierj** [jack-and-the-beans/buddies-ios#8]
1. Go to login screen
1. Tap "Already have an account"
1. Tap "Continue with Facebook"

What seemed relevant from the logs:

```
2019-01-31 22:04:24.480319-0500 Buddies[16387:1816717] -canOpenURL: failed for URL: "fbauth2:/" - error: "The operation couldn’t be completed. (OSStatus error -10814.)"
2019-01-31 22:04:24.485649-0500 Buddies[16387:1816717] -[Buddies.LoginExistingVC facebookLogin:]: unrecognized selector sent to instance 0x7fda3bf4bbd0
2019-01-31 22:04:24.491740-0500 Buddies[16387:1816717] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[Buddies.LoginExistingVC facebookLogin:]: unrecognized selector sent to instance 0x7fda3bf4bbd0'
*** First throw call stack:
(
	0   CoreFoundation                      0x000000010ff491bb __exceptionPreprocess + 331
	1   libobjc.A.dylib                     0x000000010f4e7735 objc_exception_throw + 48
	2   CoreFoundation                      0x000000010ff67f44 -[NSObject(NSObject) doesNotRecognizeSelector:] + 132
	3   UIKitCore                           0x000000011594fb4a -[UIResponder doesNotRecognizeSelector:] + 287
	4   CoreFoundation                      0x000000010ff4ded6 ___forwarding___ + 1446
	5   CoreFoundation                      0x000000010ff4fda8 _CF_forwarding_prep_0 + 120
	6   UIKitCore                           0x0000000115922ecb -[UIApplication sendAction:to:from:forEvent:] + 83
	7   UIKitCore                           0x000000011535e0bd -[UIControl sendAction:to:forEvent:] + 67
	8   UIKitCore                           0x000000011535e3da -[UIControl _sendActionsForEvents:withEvent:] + 450
	9   UIKitCore                           0x000000011535d31e -[UIControl touchesEnded:withEvent:] + 583
	10  UIKitCore                           0x000000011595e0a4 -[UIWindow _sendTouchesForEvent:] + 2729
	11  UIKitCore                           0x000000011595f7a0 -[UIWindow sendEvent:] + 4080
	12  UIKitCore                           0x000000011593d394 -[UIApplication sendEvent:] + 352
	13  UIKitCore                           0x0000000115a125a9 __dispatchPreprocessedEventFromEventQueue + 3054
	14  UIKitCore                           0x0000000115a151cb __handleEventQueueInternal + 5948
	15  CoreFoundation                      0x000000010feae721 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 17
	16  CoreFoundation                      0x000000010feadf93 __CFRunLoopDoSources0 + 243
	17  CoreFoundation                      0x000000010fea863f __CFRunLoopRun + 1263
	18  CoreFoundation                      0x000000010fea7e11 CFRunLoopRunSpecific + 625
	19  GraphicsServices                    0x0000000114e9d1dd GSEventRunModal + 62
	20  UIKitCore                           0x000000011592181d UIApplicationMain + 140
	21  Buddies                             0x000000010abfb737 main + 71
	22  libdyld.dylib                       0x0000000111dba575 start + 1
	23  ???                                 0x0000000000000001 0x0 + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

**Comment by JakeThurman** [comment]
I just saw this too :(

**Comment by lukemeierj** [comment]
Closed with PR #9
